ballerina-tests/tests/service_native_endpoint_test.bal

```diff
@@ -87,13 +83,3 @@ function testGetHostName() {
         test:assertFail(msg = "Found unexpected output type: " + response.message());
     }
 }
-
-@test:Config {}
-function testGetNoHostName() {
-    var response = serviceEndpointClient->get("/serviceEndpointHello/nohost", targetType = string);
-    if (response is string) {
-        test:assertEquals(response, "nohost", msg = "Found unexpected output");
-    } else {
-        test:assertFail(msg = "Found unexpected output type: " + response.message());
-    }
-}
```

Comment:
Any reason for removing this ?

Reply:
This test case covers a scenario where a `http:Caller` is instantiated manually. But, in real world `http:Caller` should only be instantiated internally by HTTP package. And it should have a package-private `init` function. So, this test case itself is bit unusual.